src/components/formElements/FormElements.tsx

```diff
@@ -55,3 +60,64 @@ export const CollapsableFormElementDescription = (props: { title: string; childr
         </div>
     );
 };
+
+export const PeriodeForm = (props: {
+    containerClassname?: string;
+    fraOgMed: {
+        id?: string;
+        value?: Nullable<Date>;
+        minDate?: Nullable<Date>;
+        maxDate?: Nullable<Date>;
+        setFraOgMed: (date: Nullable<Date>) => void;
+        error?: FieldError;
+        size?: 'S' | 'L';
+    };
+    tilOgMed: {
+        id?: string;
+        value?: Nullable<Date>;
+        minDate?: Nullable<Date>;
+        maxDate?: Nullable<Date>;
+        setTilOgMed: (date: Nullable<Date>) => void;
+        error?: FieldError;
+        size?: 'S' | 'L';
+    };
+}) => {
+    const { formatMessage } = useI18n({ messages: nb });
+
+    return (
+        <div className={props.containerClassname ?? styles.periodeFormContainer}>
```

Comment:
Hmm, tanken er god, men jeg synes det er litt problematisk med en gang man unnviker fra en "regel". Hvis man nå skal lage en form med react-hook-form i koden vår, så må man huske på at man alltid skal ha `<Controller` _med unntak av_ når man bruker PeriodeForm. Det er lett at sånt sklir litt ut, og den unntak-av-listen vokser litt ukontrollert.

Reply:
Men hvorfor får man ikke brukt `<Controller/>` egentlig?

Reply:
Vi kan rent teknisk sende controller med props. Problemet blir litt av å type hva controller skal være. Jeg har enda ikke funnet ut hvordan vi kan få det slik at hvis en controller har {fraOgMed: X; tilOgMed: X}, så kan <PeriodeForm /> ta den inn. Det gjør det vanskelig å generalisere det. Pga dette, så kan man ikke bruke controller, og det derfor faller vi litt ut av standaren. 

Det blir trade-offs her. Jeg ser også for meg at vi kommer til å få flere vilkår som trenger en <PeriodeForm />. vi sparer en del linjer på dette. Men ja, blir et spørsmål om vi vil ha litt mindre kode vs opprettholde regler

Reply:
_Jeg_ foretrekker å ha ting konsistent, men her er det bare å komme med egne meninger selvsagt :D Man kunne klart å gjøre den generic vil jeg tro, men det gjør nok bare at den blir litt komplisert. I så fall tror jeg det er like lettlest og håndterbart å bare bruke to DatePickers i steden for Periode. Men liker uansett tanken på å generalisere slike gjenbrukte komboer. Kan være vi får til en smud løsning med litt iterering her!

Reply:
Sjekk [her](https://github.com/navikt/su-se-framover/pull/1900) @RamziAbuQassim . Lagde et forslag på hvordan man kan få det til. Den viktige endringen er at man da må ha `fraOgMed` og `tilOgMed` i et eget object (periode). Hva tenker du om det? 😊